src/main/java/dk/bec/polonez/reservationsystem/service/DiscountService.java

```diff
@@ -16,4 +23,30 @@ public DiscountService(DiscountRepository discountRepository) {
     public int getByCode(String code){
```

Comment:
**advice** Shouldn't the method ```getByCode``` return ```Discount``` object instead of ```int```?

Reply:
yes, it is not ready yet